src/droplet.F

```diff
@@ -593,19 +641,21 @@ subroutine droplet_driver(dt,dbldt,mtime,xh,uh,ruh,xf,yh,vh,rvh,yf,zh,mh,rmh,zf,
     ! number of holes in "pdata" to fill after the arrival droplets have
     ! been assigned a location
     numBackfill = sum(Depart) + num_fallout - sum(Arrive) 
-
-    ! Determine which droplets to relocated from the end of 
-    ! the "pdata" array to the remaining holes
-    if (numBackfill .gt. 0) then 
+     ! Determine which droplets to relocated from the end of 
+     ! the "pdata" array to the remaining holes
+     if (numBackfill .gt. 0) then 
        allocate(backfill_ind(numBackfill))
+       !$acc enter data create(backfill_ind)
        if ((nparcelsLocalActive - numBackfill) .eq. 0) then
           ! Set backfill_ind to 1 to handle an exception case, 
           ! where all the active droplets fall out or leave
           ! the current MPI domain
+          !$acc parallel loop gang vector default(present)
           do i = 1, numBackfill
              backfill_ind(i) = 1
           end do
        else
+          !$acc parallel num_gangs(1) num_workers(1) default(present)
```

Comment:
Probably using `!$acc serial default(present)` is more straightforward?

Reply:
Just changed the source code.